Solutions/CiscoUmbrella/Data Connectors/CiscoUmbrella_TestingInstructions.docx

```diff
@@ -0,0 +1,112 @@
+# Testing Instructions: Cisco Umbrella Data Connector with Private Storage Account
+
+## Overview
+This document provides step-by-step instructions for testing the Cisco Umbrella data connector deployment with private storage account configuration.
+
+## Prerequisites
+Before beginning the deployment, ensure you have:
+- An Azure subscription with permissions to create resources
+- A Cisco Umbrella account with permissions to access the S3 bucket
+- Required information:
+  * Cisco Umbrella S3 bucket name
+  * AWS Access Key ID with permissions to read from the S3 bucket
+  * AWS Secret Access Key
+  * Log Analytics Workspace ID and Key
+
+## Deployment Steps
+
+### 1. Repository Setup
+```bash
+git clone https://github.com/AmudaPalani/Azure-Sentinel.git
```

Comment:
Hi @AmudaPalani,
It should be the master repository instead of your personal repository. Thanks

Reply:
I forked master repo. And then created a branch. If you want me to do it in master repo, I need access to create a branch.